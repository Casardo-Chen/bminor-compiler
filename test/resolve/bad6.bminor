// wrong scope
doSomething: function void () ={
    localVar: integer = 10;
}

main: function integer() ={
    doSomething();
    localVar++; 
    return 0;
}