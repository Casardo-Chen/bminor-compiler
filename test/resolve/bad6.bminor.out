doSomething resolves to global doSomething
resolve error: localVar is not defined.
1 resolve errors found.
